contracts/templates/NFTCollection.sol

```diff
@@ -79,28 +79,28 @@ contract NFTCollection is ERC721, ERC2981, AccessControl, Initializable {
      *************/
 
     /// Contract version, semver-style uint X_YY_ZZ
-    uint256 public constant VERSION = 1_02_00;
+    uint256 public constant VERSION = 1_02_02;
 
     /// Admin role
     bytes32 public constant ADMIN_ROLE = keccak256("ADMIN_ROLE");
 
     // Basis for calculating royalties.
     // This has to be 10k for royaltiesBps to be in basis points.
-    uint16 constant ROYALTIES_BASIS = 10000;
+    uint16 public constant ROYALTIES_BASIS = 10000;
 
     /********************
      * Public variables *
      ********************/
 
-    /// The number of currently minted tokens
+    /// The number of tokens remaining in the reserve
     /// @dev Managed by the contract
-    uint256 public totalSupply;
```

Comment:
How to get totalSupply now?

Reply:
It's still there but `totalSupply` is implemented in ERC721A